### PR TITLE
Add Inventory Reference

### PR DESCRIPTION
**Pull request by CommanderFoo**
Cut down on the example explanation as it was too much like a tutorial, and if the change event doesn't get fixed for the client, might need to add a workaround.

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
         "POIs",
         "Qbel",
         "Rasm",
+        "Raycast",
```
Comment (DoubleA-MG) [inline cspell.json:87]: What is this for?

File: src/references/inventory.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+Creators can create an item asset from **Project Content** by right-clicking and selecting **Create Item Asset** from the menu, or selecting **Create Item Asset** from the **Script** ![Script Icon](../img/EditorManual/icons/Icon_Script.png){: .image-inline-text .image-background style="background-color: #17A2E7" } dropdown.
+
+To view created item assets, select **My Items** in **Project Content**.
+<!-- vale Manticore.FirstPerson = Yes -->
+
+![!Create Item Asset](../img/Inventory/create_item_asset.png){: .center loading="lazy" }
+
+### Item Asset Properties
+
+**Item Assets** have properties that describe the item
```
Comment (DoubleA-MG) [inline src/references/inventory.md]: missing period

**Review by DoubleA-MG — CHANGES_REQUESTED** [review]
I like the example, pretty simple and clear cut.

File: cspell.json (hunk 1 above)
Comment (CommanderFoo) [reply cspell.json:87]: It's basically a dictionary of words that the spelling extension should ignore. So certain words from the API that are correct can be added it.

**Pull request merged**